### PR TITLE
new package: sincei

### PR DESCRIPTION
**Pull request by vivekbhr**
Name of the tool: sincei

Short description: A user-friendly toolkit for QC, counting, clustering and plotting of single-cell (epi)genomics data 

How does the package use scverse data structures (please describe in a few sentences): stores information in anndata format

-   [x] The code is publicly available under an [OSI-approved](https://opensource.org/licenses/alphabetical) license
-   [x] The package provides versioned releases
-   [x] The package can be installed from a standard registry (e.g. PyPI, conda-forge, bioconda)
-   [x] The package uses automated software tests and runs them via continuous integration (CI)
-   [x] The package provides API documentation via a website or README
-   [x] The package uses scverse datastructures where appropriate (i.e. AnnData, MuData or SpatialData and their modality-specific extensions)
-   [x] I am an author or maintainer of the tool and agree on listing the package on the scverse website

### Recommended

-   [x] Please announce this package on scverse communication channels (zulip, discourse, twitter)
-   [x] Please tag the author(s) these announcements. Handles (e.g. `@scverse_team`) to include are:
    -   Twitter: @vivekbhr
    -   Zulip:
    -   Discourse:
    -   Mastodon:
-   [x] The package provides tutorials (or "vignettes") that help getting users started quickly
-   [ ] The package uses the [scverse cookiecutter template](https://github.com/scverse/cookiecutter-scverse).

### CODE REVIEW

**Comment by grst** [comment]
Hi @vivekbhr, 

thanks for submitting the package! Looks good overall, I have a few specific comments

> The package can be installed from a standard registry (e.g. PyPI, conda-forge, bioconda)

- The package is available from PyPI - why do you recommend to install the stable version via `pip install git+...` in the README? 

> The package uses scverse datastructures where appropriate (i.e. AnnData, MuData or SpatialData and their modality-specific extensions)

Here I see some room for improvement... In particular, the API is not very consistent here. 

 * [GLMPCA](https://sincei.readthedocs.io/en/latest/content/modules/sincei.html#module-sincei.GLMPCA) doesn't take an AnnData object, but rather matrix
 * [TOPICMODEL](https://sincei.readthedocs.io/en/latest/content/modules/sincei.html#module-sincei.TopicModels) takes a sparse matrix and a list of cell indices, while it could take AnnData + a column name in anndata.obs
 * [multimodalClustering](https://sincei.readthedocs.io/en/latest/content/modules/sincei.html#module-sincei.multimodalClustering) does take AnnData objects already. Since it deals with multimodal data, it would be natural to support MuData input. It also returns anndata objects and umap coordinates separately, while it would be nice to write umap coordinates in .obsm and cluster labels in .obs for plotting with scanpy. 
 
It is completely fine to have a more scikit-learn like interface that works on matrices rather than anndata objects, as long as it is shown in the tutorial how this works with scverse data structures, e.g. 

```python
adata.obsm["glmpca"] = GLMPCA.fit_transform(adata.layers["raw_counts"])
```

But my criticism here is that (a) it's not consistent and (b) the Python tutorial doesn't embrace scverse data structures at all. 

>  The package provides API documentation via a website or README

There's a lot of functions in the API documentation that are not fully documented (no description on the arguments and the return values). If they are not meant to be user-facing, consider making them private and removing them from the documentation. If they are meant to be used, please document them.

**Comment by vivekbhr** [comment]
@grst Thank you for the review. Indeed, the python API was not designed to read/write anndata objects. But the CLI wrappers do store everything in anndata. I agree we can improve the tutorial to indicate how to use anndata input, like in your example.

We will work on the improvements and get back to you.

**Comment by grst** [comment]
Hi @vivekbhr ,

thanks for the update! I'll be on vacation the next three weeks and can take a look once I'm back. Or maybe @mikkelnrasmussen has a chance to review in the meanwhile?

**Comment by grst** [comment]
Ah, sorry about the GLMPCA, I missed that .fit() accepts an anndata object now. 
Then the remainin requests are about the documentation: 
 * remove unused/internal stuff from the API doc
 * provide instructions on how to use the Python API, e.g. by adding another tutorial, or by structuring the API docs a bit better

**Comment by vivekbhr** [comment]
@grst We have made substantial changes to the doc and made the python API documentation much clearer and clean. Could you have another look? I hope all the changes are there now to welcome sincei to scVerse!
